engine_test.go

```diff
@@ -1081,7 +1081,7 @@ func TestSessionVariables(t *testing.T) {
 	session := sql.NewBaseSession()
 	ctx := sql.NewContext(context.Background(), sql.WithSession(session), sql.WithPid(1))
 
-	_, _, err := e.Query(ctx, `set autocommit=1, sql_mode = concat(@@sql_mode,',STRICT_TRANS_TABLES')`)
+	_, _, err := e.Query(ctx, `set autocommit=ON, sql_mode = concat(@@sql_mode,',STRICT_TRANS_TABLES')`)
```

Comment:
autocommit=1 should work too, so I would prefer to do not change this test.

Reply:
ok, I'll add another test.